model_card_toolkit/model_card.py

```diff
@@ -489,14 +489,14 @@ def to_json(self) -> str:
                     ] = json_utils.get_latest_schema_version()
     return json_lib.dumps(model_card_dict, indent=2)
 
-  def from_json(self, json_dict: Dict[str, Any]) -> None:
+  def merge_from_json(self, json: Union[Dict[str, Any], str]) -> 'ModelCard':
```

Comment:
I moved the definition of `from_json` below `merge_from_json`, so this diff looks strange.